api/src/test/resources/answer-challenge-identify-first-response.json

```diff
@@ -0,0 +1,229 @@
+{
+  "version": "1.0.0",
+  "stateHandle": "02_SASNV3rEhVtB1v3sW5dkSvDCproFFJLi2p6ecer",
+  "expiresAt": "2021-05-17T15:20:45.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": ["create-form"],
+        "name": "select-authenticator-enroll",
```

Comment:
We don't support a lot of these authenticators in our SDK right now. Probably worth configuring for what we support. Thoughts @arvindkrishnakumar-okta?

Reply:
The list of authenticators updated